test/issuer/default_http_options.test.js

```diff
@@ -9,7 +9,7 @@ describe('Issuer#defaultHttpOptions', function () {
 
   it('includes a user-agent by default', function () {
     expect(Issuer.defaultHttpOptions).to.have.nested.property('headers.User-Agent')
-      .to.match(/^openid-client/);
+      .to.match(/openid-client/);
```

Comment:
?

Reply:
The user-agent string is derived from the package name which was previously 'openid-client' and it is now @gmsllc/openid-client